ElementX/Sources/Screens/ComposerToolbar/View/ComposerToolbar.swift

```diff
@@ -105,31 +105,32 @@ struct ComposerToolbar: View {
                 .padding(.leading, 7)
         }
     }
+    
+    private var mainContentLayout: some Layout {
```

Comment:
This name is maybe better (my bad)
```suggestion
    private var topBarLayout: some Layout {
```

Reply:
👍